scalability_and_performance/using-cluster-loader.adoc

```diff
@@ -0,0 +1,46 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+[id='using_cluster_loader{context}']
+= Using cluster loader
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context:
+
+toc::[]
+
+
```

Comment:
On the preview I'm seeing a strange thin grey horizontal rule immediately above this paragraph. Do you see it?

Reply:
Yeah, it marks the end of the TOC

Reply:
Can we get a `<br><br>` or something between?

Reply:
Yeah, I can play with it

Reply:
Thanks :smiley:

Reply:
@sjug Bummer. :disappointed: Adding extra line breaks doesn't fix the issue. It may be a thing with the templates. I'll see if there's a way to improve the way this renders in this and other topics. Raising this in a docs team meeting. :smiley: